features/step_definitions/package_json_steps.coffee

```diff
@@ -4,11 +4,13 @@ path = require 'path'
 
 module.exports = ->
 
-  @Given /^I am in a node project$/, (done) ->
-    fsExtra.outputJson path.join(@tmpDir, 'package.json'), name: '123', done
+  @Given /^I work on a Node.js project$/, (done) ->
```

Comment:
Okay, so please let me know what you want to do here. In my experience, many Node projects use package.json for dependency management (installing dependencies), even if they themselves don't get published to NPM. Mycha integrates with package.json for the same reason, to install its dependencies. I think that's the use case we want to describe here.